modules/release.js

```diff
@@ -49,11 +49,11 @@ const getNextVersion = async ({ repo, stage = 'dev' }) => {
 
 const isPre = ({ repo, stage = 'dev' }) => !isVersioned(repo) && stage.toLowerCase() === 'dev'
 
-const formatRepoStage = ({ repo, stage }) => isVersioned(repo) ? repo : `${repo} (${stage || 'unknown stage'})}`
+const formatRepoStage = ({ repo, stage }) => isVersioned(repo) ? repo : `${repo} (${stage || 'unknown stage'})`
 
 const worker = async ({ repo, stage = 'dev', response_url }) => {
   const r = {
-    response_type: 'ephemeral',
+    response_type: 'in_channel',
```

Comment:
does it work well with the `replace_original: true` on line 79?

Reply:
just saw this. looks like this param is deprecated by slack - atleast according to [this](https://api.slack.com/legacy/interactive-messages)